test/cell_test.rb

```diff
@@ -56,20 +56,20 @@ def test_fired_upon_starts_false
   end
 
   def test_fire_upon_lowers_ship_health_by_one
-    @cell.place_ship(@submarine)
+    @cell.place_ship(@cruiser)
     @cell.fire_upon
+
     assert_equal 1, @cell.ship.health
-    assert_equal 2, @cell.ship.health
   end
 
   def test_cell_has_not_been_fired_upon
-    assert_equal false, @cell.fired_upon?
-    @cell.fire_upon
-    assert_equal true, @cell.fired_upon?
+    @cell.place_ship(@cruiser)
+    @cell_2.place_ship(@submarine)
```

Comment:
This test doesn't need us to place ships; you might have been thinking of the next one. To make this one dynamic, we would repeat the tests on the old lines 66-68 but on @cell_2.